sqlmesh/cli/main.py

```diff
@@ -955,6 +965,11 @@ def table_name(obj: Context, model_name: str, dev: bool) -> None:
     default=False,
     help="If set, existing models are overwritten with the new DLT tables.",
 )
+@click.option(
+    "--dlt-path",
+    type=str,
+    help="The directory where the DLT pipeline resides. Use alongside template: dlt",
```

Comment:
[Nit] Is the template bit relevant for this command?
```suggestion
    help="The directory where the DLT pipeline resides.",
```

Reply:
Yes its not thank you removed it